manage/app/gql/queries/lead-report/email-metrics.graphql

```diff
@@ -2,6 +2,8 @@
 
 query LeadReportEmailMetrics($hash: String!, $sort: ReportEmailMetricsSortInput!, $starting: Date, $ending: Date) {
   reportEmailMetrics(hash: $hash, sort: $sort, starting: $starting, ending: $ending) {
+    showAdvertiserCTOR
+    showTotalAdClicksPerDay
```

Comment:
Return the campaign here, instead of these two fields on their own. Make sure to always include the `_id` or `id` fields where possible in order to take advantage of client-side query caching.
```graphql
reportEmailMetrics(...) {
  campaign {
    id
    showAdvertiserCTOR
    showTotalAdClicksPerDay
  }
  deployments {
    # ...
  }
```